### PR TITLE
[5.0] Add tempest filters based on services (SOC-9799)

### PR DESCRIPTION
**Pull request by stefannica**
(backport of #2180)

This change adds filters for running tempest for specific services.
Those filters should be used together with the tests2skip script which
outputs a regex to run specific tests.

All tests on the filters should be passing.

(cherry picked from commit 4e430ff4106f80ca535aeb5965323ce97d223c23)

### CODE REVIEW

File: chef/cookbooks/tempest/files/default/run_filters/magnum.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
++magnum_tempest_plugin.*
```
Comment (gosipyan) [inline chef/cookbooks/tempest/files/default/run_filters/magnum.txt]: for SOC8 and SOC7 use magnum.tests.functional.api.*

File: chef/cookbooks/tempest/files/default/run_filters/lbaas.txt [hunk 2]
```diff
@@ -0,0 +1 @@
++neutron_lbaas.*
```
Comment (gosipyan) [inline chef/cookbooks/tempest/files/default/run_filters/lbaas.txt]: need to be skipped due to http://git.suse.provo.cloud/cgit/ardana/tempest-ansible/tree/roles/tempest/files/run_filters/lbaas.txt.j2?h=stable/pike#n25
-neutron_lbaas.tests.tempest.v2.ddt.test_health_monitor_admin_state_up.CreateHealthMonitorAdminStateTest.test_create_health_monitor_with_scenarios

**Comment by gosipyan** [comment]
ceilometer filter need to be added.
+ceilometer.*

File: chef/cookbooks/tempest/files/default/run_filters/heat.txt [hunk 3]
```diff
@@ -0,0 +1 @@
++heat_tempest_plugin.*
```
Comment (gosipyan) [inline chef/cookbooks/tempest/files/default/run_filters/heat.txt]: +heat_integrationtests.*

File: chef/cookbooks/tempest/files/default/run_filters/keystone.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
++tempest.api.identity.*
++keystone_tempest_plugin.*
```
Comment (gosipyan) [inline chef/cookbooks/tempest/files/default/run_filters/keystone.txt]: keystone_tempest_plugin.* not avaible on SOC8

File: chef/cookbooks/tempest/files/default/run_filters/manila.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
++manila_tempest_tests.tests.*
+
+# bsc#1137262
```
Comment (stefannica) [inline chef/cookbooks/tempest/files/default/run_filters/manila.txt]: This has been fixed in SLES12_SP2 about 1 week ago. We need to re-enable these tests. We also need to remove the blacklist entries in cloud 9 (maybe do both as a different set of PRs after this is merged).

**Pull request merged**